Makefile

```diff
@@ -78,7 +88,7 @@ run: welcome githooks-commit install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
 node-version:
-	@NPM_GLOBAL_ROOT=$(shell $(NPM) -g root) $(BIN)/check-node-version
+	@$(BIN)/check-node-version
```

Comment:
Another interesting tidbit - `semver` is not installed (globally, or locally, of course) on Windows as part of a standard nodejs install. So you need to explicitly install it (it's in the instructions I've written as part of this PR). I've seen a discussion about how to pull in this dependency, but nothing better than the current approach struck my fancy. And since `semver` is available (always?) on Linux/OS X, it's not _that_ important to address this issue. Especially, when the error message (missing module `semver`) is a standard one and even if the user did not RTFM, he/she should be able to figure out that `semver` has to be installed.
...or have I missed a better way to do this? :D